src/main/java/io/anserini/qa/passage/IdfPassageScorer.java

```diff
@@ -75,10 +85,17 @@ public void score(String query, Map<String, Float> sentences) throws Exception {
     HashSet<String> questionTerms = new HashSet<>(Arrays.asList(question.toString()
             .trim().toLowerCase().split("\\s+")));
 
+
+    EnglishAnalyzer ea2 = new EnglishAnalyzer(CharArraySet.EMPTY_SET);
+    QueryParser qp2 = new QueryParser(LuceneDocumentGenerator.FIELD_BODY, ea2);
+    Query questionWithStopWords = qp2.parse(escapedQuery);
+    HashSet<String> questionTermsIDF = new HashSet<>(Arrays.asList(questionWithStopWords.toString()
+            .trim().toLowerCase().split("\\s+")));
+
     // add the question terms to the termIDF Map
-    for (String questionTerm : questionTerms) {
+    for (String questionTerm : questionTermsIDF) {
       try {
-        TermQuery q = (TermQuery) qp.parse(questionTerm);
+        TermQuery q = (TermQuery) qp2.parse(questionTerm);
         Term t = q.getTerm();
```

Comment:
No, `questionTerm` would just be a string. The reason each term is parsed is because `similarity.idf()` expects the term to be an instance of `Term`